istioctl/cmd/root_test.go

```diff
@@ -80,3 +80,14 @@ func TestHideInheritedFlags(t *testing.T) {
 	checkHelpForFlag(t, got, parentFlag1, false)
 	checkHelpForFlag(t, got, childFlag2, true)
 }
+
+func TestDeprecateCommand(t *testing.T) {
```

Comment:
table test format is preferred.

https://github.com/golang/go/wiki/TableDrivenTests